config.py

```diff
@@ -1,84 +1,123 @@
+STORAGE_GPU_SIZE = get_float_env("STORAGE_GPU_SIZE", 0.1) if DEVICE == "cuda" else 0
+BUFFER_GPU_SIZE = get_float_env("BUFFER_GPU_SIZE", 0.02) if DEVICE == "cuda" else 0
+TRAINER_GPU_SIZE = get_float_env("TRAINER_GPU_SIZE", 0.2) if DEVICE == "cuda" else 0
 
 ### TIME RELATED VALUES ###
-ACTIONS_PER_TURN = int(environ.get("ACTIONS_PER_TURN", 15))
-CONCURRENT_GAMES = int(environ.get("CONCURRENT_GAMES", 2))
-NUM_PLAYERS = int(environ.get("NUM_PLAYERS", 8))
-
-AUTO_BATTLER_PERCENTAGE = int(environ.get("AUTO_BATTLER_PERCENTAGE", 0))
-
-# Putting this here so that we don't scale the policy by a multiple of 5
-# Because we calculate the loss for each of the 5 dimensions.
-# I'll add a mathematical way of generating these numbers later.
-DEBUG = eval(environ.get("DEBUG", True))
-CHECKPOINT_STEPS = int(environ.get("CHECKPOINT_STEPS", 100))
-
-STARTING_EPISODE = int(environ.get("STARTING_EPISODE", 0))
-
+ACTIONS_PER_TURN = get_int_env("ACTIONS_PER_TURN", 15)
+CONCURRENT_GAMES = get_int_env("CONCURRENT_GAMES", 1)
+NUM_PLAYERS = get_int_env("NUM_PLAYERS", 8)
+AUTO_BATTLER_PERCENTAGE = get_int_env("AUTO_BATTLER_PERCENTAGE", 0)
+DEBUG = get_bool_env("DEBUG", "True")
+CHECKPOINT_STEPS = get_int_env("CHECKPOINT_STEPS", 100)
+STARTING_EPISODE = get_int_env("STARTING_EPISODE", 0)
+
+# Champion and Item Dimensions
 CHAMPION_ACTION_DIM = [5 for _ in range(58)]
 CHAMPION_LIST_DIM = [2 for _ in range(58)]
 ITEM_CHOICE_DIM = [3 for _ in range(10)]
 CHAMP_DECIDER_ACTION_DIM = CHAMPION_ACTION_DIM + [2] + ITEM_CHOICE_DIM
 
-DISCOUNT = float(environ.get("DISCOUNT", 0.997))
+DISCOUNT = get_float_env("DISCOUNT", 0.997)
 TRAINING_STEPS = 1e10
-OBSERVATION_SIZE = int(environ.get("OBSERVATION_SIZE", 10432))
-OBSERVATION_TIME_STEPS = int(environ.get("OBSERVATION_TIME_STEPS", 4))
-OBSERVATION_TIME_STEP_INTERVAL = int(environ.get("OBSERVATION_TIME_STEP_INTERVAL", 5))
+OBSERVATION_SIZE = get_int_env("OBSERVATION_SIZE", 10432)
+OBSERVATION_TIME_STEPS = get_int_env("OBSERVATION_TIME_STEPS", 4)
+OBSERVATION_TIME_STEP_INTERVAL = get_int_env("OBSERVATION_TIME_STEP_INTERVAL", 5)
 INPUT_TENSOR_SHAPE = np.array([OBSERVATION_SIZE])
-CHAMP_ENCODING_SIZE = int(environ.get("CHAMP_ENCODING_SIZE", 26))
-ACTION_ENCODING_SIZE = int(environ.get("ACTION_ENCODING_SIZE", 1045))
-ACTION_CONCAT_SIZE = int(environ.get("ACTION_CONCAT_SIZE", 83))
+CHAMP_ENCODING_SIZE = get_int_env("CHAMP_ENCODING_SIZE", 26)
+ACTION_ENCODING_SIZE = get_int_env("ACTION_ENCODING_SIZE", 1045)
+ACTION_CONCAT_SIZE = get_int_env("ACTION_CONCAT_SIZE", 83)
 ACTION_DIM = [7, 38, 38]
-# 57 is the number of champions in set 4. Don't want to add an import to the STATS in the simulator in a config file
 
-
-# Number of categories for each trait tier. Emperor for example has 2, no emperors or 1.
-TEAM_TIERS_VECTOR = [4, 5, 4, 4, 4, 3, 3, 3, 2, 4, 4, 4, 5, 3, 5, 2, 3, 5, 4, 4, 3, 4, 4, 4, 2, 5]
+# Team Tiers Vector - Number of categories for each trait tier. Emperor, for example, has 2: no emperors or 1.
+TEAM_TIERS_VECTOR = [

```

Comment:
Was there a reason this had to become 4 lines. This is a vector for trait break points. There is no significant difference in meaning between the different values here other than pertaining to different traits.

Reply:
No reason, just style convention so it isn't all on one line. Will change it back.